Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Person 1:");
+
```

Comment:
Reading 2 different people should be split by 1 function called twice

Reply:
changes made